### DISCUSSION TITLE
Ability to Not Print "Not Reflected" Reports of Paramminer on Output.ndjson

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [blacklanternsecurity/bbot#1329] [General]
That would be great if possible to add a config option for paramminer to not print not reflected items into output.ndjson

![image](https://github.com/blacklanternsecurity/bbot/assets/15929497/80621915-dfd8-4627-a14d-9b2373b3be67)

Thanks 🙏

**Comment by TheTechromancer** [comment]
Created issue: https://github.com/blacklanternsecurity/bbot/issues/1330
